_data/nixer_version.yml

```diff
@@ -0,0 +1,2 @@
+# Version of the latest release of Nixer plugin
+latest_release: 0.1.1.2
```

Comment:
do we need a separate file for that?

Reply:
Jekyll _data files seem better place for contents that are going to change often (like the version), in contrary to the _config.yaml which is meant for rather more static and "technical" stuff. 

Also changes in _data files are rendered immediately in development mode, while changing _config.yaml require restart to see the results.

Regarding the dedicated file for just versions - it leaves possibility to add another version field if necessary. In case something different than version comes, we can always rename the file to more generic if we wind it suitable.